src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -111,14 +111,19 @@ private Schema BuildSchemaFrom(GraphQLDocument document)
 
             var directives = new List<Directive>();
 
+            // process the schema definition first
             foreach (var def in document.Definitions)
             {
                 if (def is GraphQLSchemaDefinition schemaDef)
                 {
                     _schemaDef = schemaDef;
```

Comment:
`_schemaDef` must be set first or setting subscription resolvers don't get set properly.  Who knows maybe more stuff.